ci/wait-for.py

```diff
@@ -90,6 +90,7 @@ def internal_base_url(namespace, service, port):
     service_parser = subparsers.add_parser('Service')
     service_parser.add_argument('name', type=str)
     service_parser.add_argument('--port', '-p', type=int, default=80)
+    parser.add_argument('--endpoint', type=str, default='/healthcheck')
```

Comment:
slight preference for this to be listed with the other parser arguments

Reply:
oops, this should have been `service_parser`. I've updated it.